SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -268,27 +268,24 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	void UpdateServicesRegionFile();
 #endif
 
+public:
+	/**
+	 * The number of RPCs that can be in flight, per type. Changing this will require schema to be regenerated and
```

Comment:
nit- shouldn't this say "might" rather than "will"? Will it still regenerate even if the map is fully populated?

Reply:
Good point, will change to "may".